### DISCUSSION TITLE
Buyer persona

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#95] [Q&A]
Sonara muy tonto pero espero me puedan ayudar, ¿El buyer persona lo llenamos con nuestros datos verdad? estoy confundido por que leí y dice que es del cliente, pero te piden datos de alguien que trabaja en el proyecto, entonces me confundi

**Comment by Burstniart** [comment]
Tu buyer persona debe ser el cliente ideal que tu imaginas para el caso de abogabot, debes describirlo como si fuera una persona real
Toma como guia algunos atributos demograficos y psicograficos, el cliente ideal que edad tiene? Donde trabaja? Que le gusta hacer?

**Reply by AndresAVargas** [reply]
Okay creo que ya me diste una imagen mejor, gracias compañero

**Reply by AndresAVargas** [reply]
Pero por ejemplo, en la plantilla que nos dieron piden superior, herramientas que necesita para trabajar y dificultades, no se que poner

**Reply by AndresAVargas** [reply]
Pero por ejemplo, en la plantilla que nos dieron piden superior, herramientas que necesita para trabajar y dificultades, no se que poner

**Reply by CesarMezaB** [reply]
Te puede ayudar la siguiente plantilla para que te des una guía
https://www.hubspot.es/make-my-persona

**Reply by Burstniart** [reply]
Si usas esa plantilla puedes  imaginar que tu buyer p es contador, en su trabajo quien es su superior, usa una computadora o software especifico, a lo mejor le cuesta trabajo entender la tecnologia, o se cansa mucho, depende de como te l@ imaginas

**Reply by AndresAVargas** [reply]
entonces por ejemplo para esto de abogabot podría estar abierto a cualquier publico? lo cual no necesitaría ni superior ni herramientas, solo conocimiento de tecnología básica para usar la app?

**Reply by Burstniart** [reply]
Cualquier persona o programa que pueda entrar a la web puede usar la pagina
- practicamente todo el mundo
El publico objetivo son las personas a las que va enfocada la app y quienes es mas probable que usen los servicios de abogabot
- reduce el publico general a un porcentaje mas chico pero que es mas probable que se interese
El buyer pesona es el cliente ideal
- seria como el 1% pero tambien el mejor cliente
